documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -4020,6 +4020,12 @@ For more information, see xref:postgresql-toasted-values[toasted values].
 |Determines whether the connector generates events with transaction boundaries and enriches change event envelopes with transaction metadata. Specify `true` if you want the connector to do this.
 For more information, see xref:postgresql-transaction-metadata[Transaction metadata].
 
+|[[postgresql-property-publish-via-partition-root]]<<postgresql-property-publish-via-partition-root, `+publish.via.partition.root+`>>
+|`false`
+|Determines whether streaming changes for partitioned tables are captured at the root partition level (`true`) or at the individual partition level (`false`).
```

Comment:
I wonder for the layperson if it would be advantageous here to include some examples or be less technical. The goal here is to indicate the following:

* When set to `true`, partitioned table changes are emitted using the base table name.
* When set to `false`, partitioned table changes are emitted using the partition table name.